packages/analytics/analytics-metric-provider/README.md

```diff
@@ -23,7 +23,7 @@ Query data for metric cards and render default "golden signal" metrics cards usi
 
 ## Usage
 
-First, create a wrapper around `<MetricsProviderInternal />` in the host app.  This wrapper is in charge of feeding environment-specific properties to the metrics provider, most notably:
+First, create a wrapper around `<MetricsProvider />` in the host app.  This wrapper is in charge of feeding environment-specific properties to the metrics provider, most notably:
```

Comment:
We should no longer talk about creating a wrapper.  :)

Reply:
Oh yeah 😅